COPYING

```diff
@@ -0,0 +1,28 @@
+Copyright 2020 The project authors
```

Comment:
Did you mean for this to stay generic? If it were me, I would put "My name and contributors" or something like that.

Reply:
Well, I wanted to make this sound as inclusive as possible :smile:

Reply:
How about "Nicolas Bock and contributors"? This is your baby ;)

Reply:
Alright, you convinced me :smile: